packages/material-ui/src/NoSsr/NoSsr.test.js

```diff
@@ -5,22 +5,17 @@ import NoSsr from '@material-ui/core/NoSsr';
 
 describe('<NoSsr />', () => {
   const render = createClientRender();
-  const serverRender = createServerRender();
+  const serverRender = createServerRender({ expectUseLayoutEffectWarning: true });
 
   describe('server-side rendering', () => {
     it('should not render the children as the width is unknown', () => {
-      let wrapper;
-      expect(() => {
-        wrapper = serverRender(
-          <NoSsr>
-            <span>Hello</span>
-          </NoSsr>,
-        );
-      }).toErrorDev(
-        // Known issue due to using SSR APIs in a browser environment.
-        ['Warning: useLayoutEffect does nothing on the server'],
+      const container = serverRender(
+        <NoSsr>
+          <span>Hello</span>
+        </NoSsr>,
       );
-      expect(wrapper.text()).to.equal('');
+
+      expect(container.firstChild).to.equal(null);
```

Comment:
Argueably a clearer assertion since there's a different between an element with no text and no element.